scripts/benchmarks/scenarios-pretok.yaml

```diff
@@ -19,44 +19,96 @@
 # - this is the best approach to keep things flexible and
 #   allows to configure many different datasets
 #  - there is an option of setting tokenize is True or False
-data_processing:
-  dataset_name: yahma/alpaca-cleaned
-  chat_template: |
-    {%- for message in messages %}
-        {% if message['input'] != '' %}
-    Below is an instruction that describes a task, paired with an input that provides further context. Write a response that appropriately completes the request.
-
-        {% else %}
-    Below is an instruction that describes a task. Write a response that appropriately completes the request.
-
-        {% endif %}
-    ### Instruction:
-    {{ message['instruction'] }}
 
-        {% if message['input'] != '' %}
-    ### Input:
-    {{ message['input'] }}
-
-        {% endif %}
-    ### Response:
-    {{ message['output'] + eos_token }}
-    {% endfor %}
-  tokenize: True
+data_processing:
+    dataset_name: microsoft/orca-math-word-problems-200k
+    chat_template: |
+        {%- for message in messages %}
+        USER:
+            {% if message['question'] != '' %}
```

Comment:
if the `question` and `answer` fields are always present, there is no need for `if` clauses.